webhook.go

```diff
@@ -0,0 +1,28 @@
+package monta
+
+// WebhookRequest is a webhook request from Monta.
+type WebhookRequest struct {
+	// List of webhook entries delivered in a batch.
+	Entries []WebhookEntry `json:"entries"`
+
+	// Number of pending events left for delivery.
+	Pending int64 `json:"pending"`
+
+	// Timestamp of the request (milliseconds from the epoch of 1970-01-01T00:00:00Z).
+	Timestamp UnixTimestamp `json:"timestamp"`
+}
+
+// WebhookEntry is an entry of the webhook request.
+type WebhookEntry struct {
+	// Type of the entity.
+	EntityType string `json:"entityType"`
```

Comment:
Use `WebhookEntityType` here?
```suggestion
	EntityType WebhookEntityType `json:"entityType"`
```